src/Dodo.HttpClient.ResiliencePolicies.Tests/RetryPolicyTests.cs

```diff
@@ -35,7 +35,11 @@ public async Task Should_retry_6_times_for_two_threads_when_client_returns_503()
 		{
 			const int retryCount = 3;
 			var retrySettings =
-				new JitterRetrySettings(retryCount, medianFirstRetryDelay: TimeSpan.FromMilliseconds(50));
+				new JitterRetrySettings
+				{
+					RetryCount = retryCount,
+					SleepDurationProvider = JitterRetrySettings._defaultSleepDurationProvider(retryCount, TimeSpan.FromMilliseconds(50))
```

Comment:
It is not the same logic here as it was before. Actually we should fill absent MedianFirstRetryDelay and do not set SleepDurationProvider at all, because JutterRetrySettings will take correct provider by default. We should discuss it.